### PR TITLE
added ackley and rasrigin

### PR DESCRIPTION
**Pull request by milakis**

### CODE REVIEW

**Comment by milakis** [comment]
I am still trying to solve the problems with checks. Meanwhile, could you check the graphs? I've done them both for 500 and 100 points for x1, x2. I think Rastrigin for 500 looks fine. For Ackley I don't observe that explicit local minima. For 100 it is more explicit. 
And one more question. I've seen that Rachel added description for eoq figure. Should I add it too? I though we create these figures only to insert them to the description.

**Comment by peisenha** [comment]
Just go ahead and finish everything that you think it is in good shape and then I will be happy to give feedback then. Regarding the figures, a little sentence that describes what one sees is probably in order.

---

Philipp Eisenhauer
Economist

Mail [peisenha@protonmail.com](mailto:eisenhauer@policy-lab.org)
Web www.eisenhauer.io
Repository https://github.com/peisenha

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by loikein** [comment]
Oh my word, I overlooked this. Will work on it tomorrow…

**Comment by loikein** [comment]
It's a couple of commits behind `master` now, so I think it's a good idea to fetch it first and then look at what can be done. Commits on the way…

**Comment by loikein** [comment]
Looks good to me now. Please take over @milakis

File: docs/_static/codes/fig-ackley.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+xvalues2 = xvalues[:, 1]
+
+fig = plt.figure()
+ax = Axes3D(fig)
+ax.scatter(xvalues1, xvalues2, yvalues, c=yvalues, cmap="viridis", linewidth=0.05)
+ax.set_xlabel("$x_1$")
+ax.set_ylabel("$x_2$")
+ax.set_zlabel("$f(x_1, x_2)$")
+ax.set_title("Ackley function")
+fig.savefig("fig-ackley-500")
```
Comment (peisenha) [inline docs/_static/codes/fig-ackley.py:38]: Please remove 500 from the filename extension and do not set a title.

**Pull request merged**